packages/api/models/granules.js

```diff
@@ -364,7 +365,7 @@ class Granule extends Manager {
       files,
       error,
       published,
-      createdAt: workflowStartTime,
+      createdAt: createdAt || workflowStartTime,
```

Comment:
Can we get a unit test for this function to verify that `createdAt` is used if present otherwise it falls back? I know it's sort of tested elsewhere but I think a small unit test for this function would be worthwhile

Reply:
Done.